.circleci/config.yml

```diff
@@ -1,23 +1,47 @@
 version: 2.1
 
-orbs:
-  python: circleci/python@1.4.0
+jobs:
+  build_and_test:
+      parameters:
+        python-version:
+          type: string
+        django-version:
+          type: string
+      docker:
+        - image: cimg/python:<<parameters.python-version>>
+          environment:
+            CIRCLECI: 1
+            PGHOST: 127.0.0.1
+        - image: circleci/postgres:10-alpine-ram
+          environment:
+            POSTGRES_USER: circleci
+            POSTGRES_DB: democracy_club
+      steps:
+        - checkout
+        - run:
+            name: install dependencies
+            command: |
+              python3 -m venv .venv
+              . .venv/bin/activate
+              pip install -r requirements.txt
+              pip install django==<<parameters.django-version>>
```

Comment:
I was really confused where to put this; makes total sense now that I see it here :)